uk.co.piggz.amazfish.json

```diff
@@ -0,0 +1,154 @@
+      "buildsystem": "qmake",
+      "builddir": true,
+      "cleanup": [
+        "/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/qtmpris/archive/refs/tags/1.0.6.tar.gz",
+          "sha256": "384eac76ec7d29d1de6fa93c45007ccd506d6cc0e4c69e19d4250e16cc0f34a2"
+        },
+        {
+          "type": "patch",
+          "path": "qtmpris-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "libkdb3",
+      "buildsystem": "cmake",
+      "builddir": true,
+      "cleanup": [
+        "/include",
+        "/mkspecs"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/libraries/kdb/-/archive/tags/v3.2.0/kdb-v3.2.0.tar.gz",
+          "sha256": "3a5c660f0080c6971ac1046be84a0accd86a5296e9df4c585b1b91361b5eed20"
+        },
+        {
+          "type": "patch",
+          "path": "6.patch"
+        },
+        {
+          "type": "patch",
+          "path": "upstream_Fix-build-with-newer-Qt.patch"
+        },
+        {
+          "type": "patch",
+          "path": "libkdb3-features.patch"
+        }
+      ]
+    },
+    {
+      "name": "extra-cmake-modules",
+      "buildsystem": "cmake-ninja",
+      "cleanup": [
+         "*"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/frameworks/extra-cmake-modules/-/archive/v5.115.0/extra-cmake-modules-v5.155.0.tar.gz",
+          "sha256": "b63a9376786c29487f9b513df06b08ecda534852ae09d2e045f35e4e32dc1433"
+        }
+      ]
+    },
+    {
+      "name": "bluezqt",
+      "buildsystem": "cmake",
+      "builddir": true,
+      "cleanup": [
+        "/include",
+        "/mkspecs"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/frameworks/bluez-qt/-/archive/v5.115.0/bluez-qt-5.115.0.tar.gz",
+          "sha256": "e0c5012e2518d6ade9eefcff118b1f71b96a40a12396d07df798ce5afca7f3fc"
+        },
+        {
+          "type": "patch",
+          "path": "bluez-install-run.patch"
+        }
+      ]
+    },
+    {
+      "name": "harbour-amazfish",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "FLAVOR=qtcontrols"
+      ],
+      "build-options": {
+        "env": [
+          "QMAKEPATH=/app"
+        ]
+      },
+      "post-install": [
+        "install -Dm755 run-amazfish.sh /app/bin/run-amazfish.sh",
+        "install -Dm644 ui/icons/128x128/harbour-amazfish-ui.png /app/share/icons/hicolor/128x128/apps/uk.co.piggz.amazfish.png",
+        "install -Dm644 ui/harbour-amazfish-ui.appdata.xml /app/share/appdata/harbour-amazfish-ui.appdata.xml",
+        "sed -i 's/Exec=.*/Exec=run-amazfish.sh/' /app/share/applications/harbour-amazfish-ui.desktop"
+      ],
+
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/piggz/harbour-amazfish.git",
+          "tag": "2.2.5"
```

Comment:
this tag doesn't exist. That's the build failure.

Reply:
waiting for @piggz to merge appdata, because without that it will fail as well.

Reply:
Feel free to start a build when it does.

Reply:
I have updated the appdata.xml, and will try to rebuild after merge